src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,21 @@
+import "./hero.scss";
+
+const Hero = () => {
+  return (
+    <section className="hero__container">
```

Comment:
the first class must named "hero" 
hero__container - it's name for the hero's child

Reply:
done